x-pack/plugins/index_management/public/application/components/mappings_editor/components/document_fields/fields/create_field/required_parameters_forms/index.ts

```diff
@@ -22,10 +21,9 @@ const typeToParametersFormMap: { [key in DataType]?: ComponentType<any> } = {
   token_count: TokenCountTypeRequiredParameters,
   scaled_float: ScaledFloatTypeRequiredParameters,
   dense_vector: DenseVectorRequiredParameters,
-  runtime: RuntimeTypeRequiredParameters,
 };
 
-export const getRequiredParametersFormForType = (
+export const getParametersFormForType = (
```

Comment:
Not sure why you had changed the name of the handler. 🤔  It correctly stated that it would return only the _required_ parameters for a field type.

Reply:
Or am I reading this the other way around?? 😄

Reply:
Yes, I clarified the name in my original PR and this revert reintroduces the original, slightly more ambiguous name.

Reply:
OK, yes your change was indeed clearer 👍